docs/examples/general/expressions/expr_examples.ipynb

```diff
@@ -4,26 +4,32 @@
    "cell_type": "markdown",
```

Comment:
<p>forgot to remove the pipeline class?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Done</p>